assets/src/edit-story/components/tabview/index.js

```diff
@@ -56,27 +57,29 @@ const Tab = styled.li.attrs(({ isActive }) => ({
   cursor: pointer;
   border: none;
   background: none;
-  color: ${({ theme }) => theme.DEPRECATED_THEME.colors.fg.white};
+  color: ${({ theme }) => theme.colors.fg.tertiary};
   font-family: ${({ theme }) => theme.DEPRECATED_THEME.fonts.tab.family};
   font-size: ${({ theme }) => theme.DEPRECATED_THEME.fonts.tab.size};
   font-weight: ${({ theme }) => theme.DEPRECATED_THEME.fonts.tab.weight};
   padding: 12px 0px;
-  margin: 0px 16px;
+  margin: 0 22px;
   margin-bottom: -1px;
   position: relative;
 
   ${({ isActive, theme }) =>
     isActive &&
     `
-    border-bottom: 1px solid ${theme.DEPRECATED_THEME.colors.accent.primary};
+    border-bottom: 2px solid ${theme.colors.fg.primary};
```

Comment:
And this one needs the padding-bottom reduced by the same 2px with: `padding-bottom: 8px`.